plugin/neomake.vim

```diff
@@ -84,13 +84,15 @@ function! s:setup_autocmds() abort
         if !exists('*nvim_buf_add_highlight')
             autocmd BufEnter * call neomake#highlights#ShowHighlights()
         endif
-        if has('timers')
-            autocmd CursorMoved * call neomake#CursorMovedDelayed()
-            " Force-redraw display of current error after resizing Vim, which appears
-            " to clear the previously echoed error.
-            autocmd VimResized * call timer_start(100, function('neomake#EchoCurrentError'))
-        else
-            autocmd CursorMoved * call neomake#CursorMoved()
+        if get(g:, 'neomake_handle_cursormoved', 1)
+            if has('timers')
+                autocmd CursorMoved * call neomake#CursorMovedDelayed()
+                " Force-redraw display of current error after resizing Vim, which appears
+                " to clear the previously echoed error.
+                autocmd VimResized * call timer_start(100, function('neomake#EchoCurrentError'))
+            else
+                autocmd CursorMoved * call neomake#CursorMoved()
+            endif
```

Comment:
maybe we should move all of there to autoload, no need to handle cursormove when neomake checker is called.

Reply:
Yes, could be done only when features are used - like I've tried with other autocommands before, e.g. the statusline.